src/components/molecules/Cv/components/Languages.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from 'react';
+import { SkillDots, CvHeading } from '../../../atoms';
+import { BoxProps, Flex, Text } from '../../../quarks';
+import { ICvLanguage } from '../Cv.types';
+import { mapLanguageLevelToDots } from '../Cv.utils';
+
+export const Languages: FC<ILanguagesProps> = ({ languages = [], ...props }) => {
+  return (
+    <>
+      <CvHeading heading="Znajomość języków obcych" />
+      {languages?.map(item => {
+        return (
+          <Flex
+            key={item.id}
+            sx={{
+              alignItems: 'center',
+              fontScale: 2,
+              mb: 2,
+            }}
+            {...props}
+          >
+            <SkillDots score={mapLanguageLevelToDots(item.level!)} />
+            <Text fontWeight="bold" pr={2}>
+              {item?.name}:{' '}
```

Comment:
jeśli zabezpieczamy się przed brakiem name / level w item to w takim przypadku braku name / level lepiej w ogóle nie wyświetlać całego item

Reply:
to zabezpieczenie nie ma sensu, usuwam ;P